Agoric-CLI.md

```diff
@@ -0,0 +1,90 @@
+## Agoric CLI
+
+You use the Agoric CLI to install dependencies, initialize, deploy, and start Agoric projects. Add one of five command names to `agoric` to specify what command to run. **Note**: Required argument names need not be given, only their values. Optional argument names must be given, along with their values. See the command's examples if you're not sure if a name is needed.
+
+In general, you want to issue Agoric CLI commands in this order:
+1. `agoric init`
+2. `agoric install`
+3. `agoric start` (Usually with `--reset`)
+4. `agoric deploy`
+
+Use `agoric help` whenever you need help with one of the above Agoric CLI commands.
+
```

Comment:
You should add the output from `agoric help`, which describes the global options that can be used. Notably, `-v` will produce more verbose output.